packages/calling/src/CallingClient/calling/callManager.ts

```diff
@@ -421,6 +429,21 @@ export class CallManager extends Eventing<CallEventTypes> implements ICallManage
   public getActiveCalls = (): Record<string, ICall> => {
     return this.callCollection;
   };
+
+  /**
+   * @param deviceId -.
+   * @param line -.
+   */
+  public updateLine(deviceId: string, line: ILine) {
+    this.lineDict[deviceId] = line;
+  }
+
+  /**
+   * @param deviceId -.
+   */
+  private getLine(deviceId: string) {
```

Comment:
Change the function name also as per what is being returned

Reply:
Changed.